soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -168,7 +165,7 @@ static bool RupeeHandler(std::shared_ptr<Ship::Console> Console, const std::vect
         return CMD_FAILED;
     }
 
-    gSaveContext.rupees = rupeeAmount;
+   gSaveContext.rupees = rupeeAmount;
```

Comment:
might be nice to make an effect for `SetRupees` that's similar to `ModifyRupees`, but that definitely doesn't need to be part of this PR and can be a future thing

Reply:
sounds good, would you mind making an issue for moving them over to using GI? (just thinking it'd be nice to have a place to jot down any complications that come to mind while this is still fresh)

Reply:
https://github.com/HarbourMasters/Shipwright/issues/2397